torch/csrc/profiler/collection.h

```diff
@@ -146,6 +148,7 @@ struct ExtraFields<EventType::TorchOp> : TorchOpBasicFields {
   uint64_t correlation_id_;
   time_t end_time_ns_;
   std::vector<op_input_t> inputs_;
+  std::vector<op_input_t> concrete_inputs_;
```

Comment:
Based on some earlier discussion 1-2 months ago, we concluded that we should separate "input dims" and "concrete inputs" in case there are things that rely on "input dims".

Does it need to be separated at this level, or could it be separated just at the level of the kineto traces? (e.g. the changes in profiler_kineto.cpp)? i.e., would it be okay to put all this info into `inputs_` or is it good to keep it separated like I did in this PR?

Reply:
We can go either way, I think the current PR looks good for now, and also your name change from inputs_ to input_shapes_ sounds looks good.